app/controllers/api/v1/education/tweets_controller.rb

```diff
@@ -0,0 +1,6 @@
+class Api::V1::Education::TweetsController < ApplicationController
+
+  def index
+    render json: Tweet.where(dataset: "education").order(post_time: :desc)
```

Comment:
This gets tweets with the search term education in which the most recent tweet is first right?